src/main/java/frc/trigon/robot/subsystems/collector/Collector.java

```diff
@@ -0,0 +1,46 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Collector extends SubsystemBase {
+
+    private static final WPI_TalonFX
+            openingMotor = CollectorConstants.OPENER_MOTOR,
+            collectingMotor = CollectorConstants.COLLECTOR_MOTOR;
+
+    private final static Collector INSTANCE = new Collector();
+
+    private Collector() {
+
+    }
+
+    public static Collector getInstance() {
+        return INSTANCE;
+    }
+
+    private void stop() {
+        collectingMotor.disable();
+        openingMotor.disable();
+    }
+
+    private void collect() {
+        openingMotor.set(CollectorConstants.OPENING_VOLTAGE);
```

Comment:
`.set` takes a percent between -1 and 1 not a voltage. if you want to set a voltage use `.setVoltage` though I don't think you need to use setVoltage and giving set a percent output is fine but if @laviRZ wants to use setVoltage then its use that.

Reply:
Move to using percent output and not voltage